tests/OrganizationsTest.php

```diff
@@ -143,6 +144,45 @@ public function testUpdateOrganization()
         }
     }
 
+    public function testOrganizationCreateWithCrmId()
+    {
+        $crmId = '1243';
+        $organization = $this->client->createOrganization($this->testMaintainerId, [
+            'name' => 'Test org',
+            'crmId' => $crmId,
+        ]);
+
+        $organization = $this->client->getOrganization($organization['id']);
+        $this->assertEquals($crmId, $organization['crmId']);
+    }
+
+    public function testMaintainerMemberCanUpdateCrmId()
```

Comment:
Tenhle test mi failuje na tom, že crmId cannot be null

Reply:
už jsem fixnul na backendu